common/http_timing.py

```
@@ -0,0 +1,103 @@
+        if "dns_start" in self.timing and "dns_end" in self.timing:
+            return self.timing["dns_end"] - self.timing["dns_start"]
+        return 0.0
+
+
+async def measure_http_request_timing(
+    session: aiohttp.ClientSession,
+    method: str,
+    url: str,
+    headers: Optional[Dict[str, str]] = None,
+    json_data: Optional[Dict[str, Any]] = None,
+    exclude_connection_time: bool = True,
```

Ruff (0.13.1)

Unused function argument: `exclude_connection_time`

(ARG001)